genlayer_py/types/transactions.py

```
@@ -413,17 +414,12 @@ def _decode_input_data(self) -> Union[DecodedDeployData, DecodedCallData, None]:
                 }
 
             else:
-                print(
-                    "[decode_input_data] Unexpected RLP array length:",
-                    len(rlp_decoded_array),
-                    rlp_decoded_array,
+                logging.warning(
+                    f"[decode_input_data] Unexpected RLP array length: {len(rlp_decoded_array)} {rlp_decoded_array}",
```

Pylint (3.3.7)

[convention] Line too long (117/100)

(C0301)